Source/WebCore/editing/AlternativeTextController.cpp

```diff
@@ -101,6 +101,10 @@ void AlternativeTextController::startAlternativeTextUITimer(AlternativeTextType
 #if ENABLE(ALTERNATIVE_TEXT_REQUIRES_AUTOMATIC_SPELLING_CORRECTION)
```

Comment:
Can we move this inside `!isAutomaticSpellingCorrectionEnabled()` to reduce code duplication?